src/main/java/org/apache/commons/compress/archivers/tar/TarConstants.java

```diff
@@ -227,6 +227,13 @@
      */
     byte   LF_OLDNORM = 0;
 
+    /**
+     * Offset inside the header for the "link flag" field.
+     *
+     * @since 1.22
+     */
+    int    LF_OFFSET = 156;
```

Comment:
No need for whitespace IMO.

Reply:
I'm just using the same style of the surrounding code. It looks better, unless I also remove spaces from everything else.